frontend/dashboard/components/RepoList/ActionLinks.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ActionLinks } from './ActionLinks';
+import { repository } from 'app-shared/mocks/mocks';
+import { textMock } from '../../../testing/mocks/i18nMock';
+
+jest.mock('../MakeCopyModal', () => {
+  return {
+    MakeCopyModal: React.forwardRef(function MockedMakeCopyModal(
+      props,
+      ref: React.Ref<HTMLDivElement>,
+    ) {
+      return <div ref={ref}>Mocked MakeCopyModal</div>;
+    }),
+  };
+});
+
+const repo = {
+  ...repository,
+  name: 'test-repo',
+  full_name: 'test-org/test-repo',
+  html_url: '/repos/test-org/test-repo',
```

Comment:
Maybe make consts for these so we can use the same consts in the assertions?

Reply:
Yes, that's tidier 😊